clang/test/Driver/sycl-oneapi-gpu-intelgpu.cpp

```diff
@@ -233,3 +233,10 @@
 // CHECK_TOOLS_BEOPTS_MIX: opencl-aot{{.*}} "-DCPU"
 // CHECK_TOOLS_BEOPTS_MIX-NOT: "-DDG1"
 // CHECK_TOOLS_BEOPTS_MIX: ocloc{{.*}} "-device" "skl"{{.*}}"-DSKL2"
+
+/// Check that target is passed to sycl-post-link for filtering
+// RUN: %clangxx -fsycl-targets=intel_gpu_pvc,intel_gpu_dg1 \
+// RUN:   -fsycl -fno-sycl-device-lib=all -fno-sycl-instrument-device-code \
```

Comment:
nit - check is pretty minimal, the `-fno-sycl-device-lib=all -fno-sycl-instrument-device-code` options are not needed.

Reply:
[Remove unnecessary flags from driver test](https://github.com/intel/llvm/pull/12727/commits/035539cc7086f71d2f26174061053d2fbf9b9f1c)